crates/re_web_viewer_server/build.rs

```diff
@@ -17,9 +17,13 @@ fn should_run() -> bool {
         // We should build the web viewer before starting crate publishing
         Environment::PublishingCrates => false,
 
-        // TODO(emilk): only build the web viewer explicitly on CI
-        Environment::RerunCI | Environment::CondaBuild => true,
+        // We build the web viewer in an explicit, separate step.
+        Environment::RerunCI => false,
 
+        // TODO(emilk): build the web viewer as an explicit step instead
+        Environment::CondaBuild => true,
```

Comment:
@jleibs do you know how this works?

Reply:
Looks like we already build the web-viewer as an explicit step:

https://github.com/conda-forge/rerun-sdk-feedstock/blob/main/recipe/build.sh#L18